server/spec/test_helper.rb

```diff
@@ -6,40 +11,50 @@
   add_filter do |source_file|
     false
   end
+end
 
-  class TestHelper < Object
+include Logging
 
-    ## return a fully qualified path to the test file directory
-    ## It assumes standard locations and looking for directory with name 'test-files'
-    def self.findTestFileDirectory
-      currentDirectory = Dir.pwd()
+class TestHelper < Object
 
-      checkDir = "#{currentDirectory}/server/test-files"
-      return checkDir if Dir.exist?(checkDir);
+  # store a global default log level that test files can
+  # look up and share.
+  @@log_level = Logger::Severity::WARN
+  #@@log_level  = Logger::Severity::DEBUG
```

Comment:
It's just kept there as an example, so someone can change the level for all tests by uncommenting a single line.